vault/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+***Fixed***:
+
+* Add metrics for PKI tidy operations ([#14327](https://github.com/DataDog/integrations-core/pull/14327))
```

Comment:
```suggestion
* Add metrics for PKI tidy operations. See [#14327](https://github.com/DataDog/integrations-core/pull/14327).
```
Small suggestion to match formatting of past entries

Reply:
The format is enforced by a CI check: https://github.com/DataDog/integrations-core/actions/runs/5858011331/job/15881104575?pr=14327